dev/breeze/tests/test_selective_checks.py

```diff
@@ -34,17 +34,39 @@
 from airflow_breeze.utils.packages import get_available_distributions
 from airflow_breeze.utils.selective_checks import (
     ALL_CI_SELECTIVE_TEST_TYPES,
-    ALL_PROVIDERS_SELECTIVE_TEST_TYPES,
     SelectiveChecks,
+    _get_test_list_as_json,
+    _split_list,
 )
 
 ANSI_COLORS_MATCHER = re.compile(r"(?:\x1B[@-_]|[\x80-\x9F])[0-?]*[ -/]*[@-~]")
 
 
 ALL_DOCS_SELECTED_FOR_BUILD = ""
 ALL_PROVIDERS_AFFECTED = ""
-LIST_OF_ALL_PROVIDER_TESTS = " ".join(
+
+ALL_CI_SELECTIVE_TEST_TYPES_AS_JSON = json.dumps(
+    [
+        {"description": "API...Serialization", "test_types": ALL_CI_SELECTIVE_TEST_TYPES},
```

Comment:
Should we add a proper description?

Reply:
> Should we add a proper description?

This description is automatically generated based on "first...last" test type and limited in length, because the description is used in generating job_id and artifact names that cannot exceed certain length. 

So .. welll. That's the best we can do for now :)